### ISSUE TITLE
[expo-updates] [SDK 49] Static images are not loaded when expo-updates is installed

### ISSUE THREAD

**Issue by devalnor** [expo/expo#24117]
### Minimal reproducible example

https://github.com/devalnor/expo-bug-001

### Summary

The presence of `expo-updates` causes production and simulator builds to not loading some static images.
I discovered this after upgrading a project to SDK 49. The issue does not happen in SDK 48.

| Observed Bug | Normal behavior (without expo-updates) ] |
|------|------|
| ![Bug](https://github.com/expo/expo/assets/226090/50ebad06-a4af-403d-a2e6-347e4fcadba4) | ![Normal behavior (without expo-update ](https://github.com/expo/expo/assets/226090/3507288b-6882-4354-a0d9-45c7c23e40a4) |

To reproduce the issue:

* Adapt the `app.json` with your account name
* run `eas update:configure` to link expo updates
* Run `npx expo run:android --variant release`

Not that you have to delete the app before to retry otherwise it will not be updated with your changes.

### Environment

```
  expo-env-info 1.0.5 environment info:
    System:
      OS: macOS 13.5
      Shell: 5.9 - /bin/zsh
    Binaries:
      Node: 18.17.0 - ~/.volta/tools/image/node/18.17.0/bin/node
      Yarn: 1.22.19 - ~/.volta/tools/image/yarn/1.22.19/bin/yarn
      npm: 9.6.7 - ~/.volta/tools/image/node/18.17.0/bin/npm
      Watchman: 2023.08.14.00 - /opt/homebrew/bin/watchman
    Managers:
      CocoaPods: 1.11.3 - /usr/local/bin/pod
    SDKs:
      iOS SDK:
        Platforms: DriverKit 22.4, iOS 16.4, macOS 13.3, tvOS 16.4, watchOS 9.4
    IDEs:
      Android Studio: Electric Eel 2022.1.1 Patch 2 Electric Eel 2022.1.1 Patch 2
      Xcode: 14.3.1/14E300c - /usr/bin/xcodebuild
    npmPackages:
      expo: ~49.0.8 => 49.0.8
      react: 18.2.0 => 18.2.0
      react-native: 0.72.4 => 0.72.4
    Expo Workflow: bare

```

**Comment by douglowder** [comment]
Most likely the issue is that your repro is missing the correct Metro config. See this comment from an earlier related issue: https://github.com/expo/expo/issues/22656#issuecomment-1624240852

**Comment by jayveenW** [comment]
This is not related to the Metro Config. I have an expo managed app, I have to revert my production app to Expo 48 as this is really a blocking issue preventing us to use images cached in expo-file-system. I don't have this issue with expo-asset, or when images are loaded from server.

**Comment by douglowder** [comment]
@jayveenW I cloned your repo and ran through your steps to reproduce the issue. I am unable to reproduce the problem. Both images appear in the release Android build running on the emulator.

**Comment by eipark** [comment]
I have encountered the same issue. Static asset image rendering fine in test flight, then once expo-updates was added, it no longer appears.

**Comment by sergioisidoro** [comment]
I'm encountering some strange behaviour also with static assets, and I'm also using expo-updates. 

Curiously enough everything works on certain devices but not others (iPhoneX works, but does not on iPhone 12) with the exact same build. Also behaviour seems to be different in simulator prebuilds and production builds.

This reminded me I also reported an issue with static assets a while ago in react-navigation repo [1], and another developer figured out it could be solved by removing assets for cerain pixel ratios [2]. There was talks of opening an issue in Expo, but I don't think that was done

I'm wondering if the extra `.` is resulting in wrong extension / resoution / filtering when bundling assets with expo updates?

[1] https://github.com/react-navigation/react-navigation/issues/11792
[2] https://github.com/react-navigation/react-navigation/pull/12019

**Comment by antoinerousseau** [comment]
This comment solved it for me: https://github.com/expo/expo/issues/22656#issuecomment-1628535353

**Comment by t0x1c123** [comment]
> This comment solved it for me: [#22656 (comment)](https://github.com/expo/expo/issues/22656#issuecomment-1628535353)

Nice! ame for me!!!! Thanks

**Comment by angel-luis** [comment]
> This comment solved it for me: [#22656 (comment)](https://github.com/expo/expo/issues/22656#issuecomment-1628535353)

not for me, where did you import the expo-asset? I'm doing in the root _layout.tsx, I don't have any app or index.js.
